src/openforms/submissions/templates/admin/submissions/submission/logs.html

```diff
@@ -81,5 +100,6 @@ <h4> {% trans "Logic rules" %} </h4>
             {% endfor %}
         {% endif %}
     {% endblock %}
+    <script src="{% static "bundles/core-js.js" %}"></script>
     <script src="/static/admin/js/collapse.js"></script>
```

Comment:
I'd swap the order of the script inclusions - our bundle is custom and should be last (it's also much larger, so the collapse can _maybe_ load a bit earlier).

I've also only now noticed you don't use the `{% static %}` tag for the collapse, so please modify that as well.